test/service/test_manifest.py

```diff
@@ -241,6 +241,30 @@ def sort_key(row: JSON) -> bytes:
         expected.sort(key=sort_key)
         self.assertEqual(expected, manifest)
 
+    def _assert_pfb(self,
+                    expected_schema: JSON,
+                    expected_entities: list[JSON],
+                    actual: Response):
+        """
+        Assert that the body of the given response contains a valid PFB manifest
+        matching the expected schema and contents, disregarding the order of the
+        PFB entities.
+
+        :param expected_schema: a PFB schema.
+
+        :param expected_entities: a list of PFB entities.
+
+        :param actual: an HTTP response containing a PFB manifest.
+        """
+        manifest = fastavro.reader(BytesIO(actual.content))
+        schema = manifest.writer_schema
+        entities = list(manifest)
+        sort_key = compose_keys(none_safe_key(), itemgetter('id'))
+        entities.sort(key=sort_key)
+        expected_entities.sort(key=sort_key)
```

Comment:
Why do we need to do this? Can we sort the cans and commit them with the expected order?